addon/components/head-layout.js

```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
+import layout from 'ember-cli-head/templates/components/head-layout';
+
+const {
+  Component,
+  get,
+  computed,
+  getOwner
+} = Ember;
```

Comment:
FWIW, since this is moved to `addon/**` we should start migrating to the newer style imports directly.

This would become:

```js
import Component from '@ember/component';
import { get } from '@ember/object';
import { computed } from '@ember/object';
import { getOwner } from '@ember/application';
```